Documentation/Communicational Patterns/1. Introduction.md

```diff
@@ -0,0 +1,67 @@
+![document type: concept](https://themomax.github.io/resources/markdown-labels/document_type_concept.svg)
```

Comment:
I would suggest to move the resources over to this repository 👍

Reply:
I would advise against hosting them right inside the Apodini/Apodini repository. Seems kind of dirty to me.

Reply:
I think it would be fine so store them in the documentation folder but I would also be fie to have a separate repo in the Apodini organization.

Reply:
They are now hosted on Apodini.github.io